renderer/reducers/app.js

```diff
@@ -63,14 +63,10 @@ export const initApp = () => async (dispatch, getState) => {
 export const terminateApp = (event, handler) => async dispatch => {
   dispatch({ type: TERMINATE_APP })
 
-  // Disconnect gRPC streams.
-  const walletUnlocker = await walletUnlockerService
-  if (await walletUnlocker.can('disconnect')) {
-    await walletUnlocker.disconnect()
-  }
-  const lightning = await lightningService
-  if (await lightning.can('disconnect')) {
-    await lightning.disconnect()
+  // Disconnect gRPC services.
+  const grpc = await grpcService
+  if (await await grpc.can('disconnect')) {
```

Comment:
wait wait wait, are you saying we need all those awaits?

Reply:
lol. Updated